Controller/PageAdminController.php

```diff
@@ -120,7 +120,7 @@ public function createAction(Request $request = null)
                 return $this->redirect($this->admin->generateUrl('create', array(
                     'siteId' => $sites[0]->getId(),
                     'uniqid' => $this->admin->getUniqid(),
-                )));
+                ) + $request->query->all()));
```

Comment:
Because it is way more concise, that's all. Plus you don't have to go back and forth to add parenthesis.